launcher/main.go

```diff
@@ -176,6 +178,19 @@ func instanceId() string {
 	return enc.EncodeToString(sum[:])
 }
 
+func getOptionValue(args []string, option string) string {
```

Comment:
I would prefer avoiding regex and inspecting the arguments of other programs if possible. I think the usual approach with environment variables set via `makeWrapper` would be better, maybe `WAYLAND_PROXY_DISPLAY`?

Reply:
If we move the whole argument generation to the launcher, we also do not need this inspection code. I will update the code accordingly.